plugins/server/io.github.damirdenis-tudor/ktor-server-rabbitmq/versions.ktor.yaml

```
@@ -1 +1 @@
-"[3.0,)": io.github.damirdenis-tudor:ktor-server-rabbitmq:1.3.+
\ No newline at end of file
+"[3.0,)": io.github.damirdenis-tudor:ktor-server-rabbitmq:1.3.3+
```

I'd suggest leaving this as is since the version will automatically match 1.3.3

```suggestion
"[3.0,)": io.github.damirdenis-tudor:ktor-server-rabbitmq:1.3.+
```

---

You're right. Thanks!